### PR TITLE
TINY-9587: Increase help dialog horizontal rule contrast in dark mode

### PR DESCRIPTION
**Pull request by ArvinJ-H**
Related Ticket: 

Description of Changes:
* Placeholder text

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [ ] Docs ticket created (if applicable)

GitHub issues (if applicable):

### CODE REVIEW

File: modules/tinymce/CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Templates will be parsed before preview and insertion to make preview consistent with inserted template content and prevent XSS. #TINY-9244
 - Pressing backspace in an empty line now preserves formatting in a previous empty line. #TINY-9454
 - Pressing enter inside the `inputfontsize` input would not move the focus back into the editor content. #TINY-9598
+- Help dialog horizontal rule now have a higher contrast in dark mode. #Tiny-9587
```
Comment (MitchCTiny) [inline modules/tinymce/CHANGELOG.md:30]:
```suggestion
- Improved contrast of table borders within dialogs in the `oxide-dark` skin. #TINY-9587
```
This also affects the wordcount dialog, but should be more specific to the location of the change.

File: modules/oxide/gulpfile.js [hunk 2]
```diff
@@ -108,7 +108,7 @@ gulp.task('monitor', function (done) {
     this.server.on('close', done);
   });
 
-  gulp.watch('./src/**/*').on('change', gulp.series('css', 'buildDemos', 'copyTinymce'));
+  gulp.watch('./src/**/*').on('change', gulp.series('css', 'buildDemos', 'buildSkinSwitcher', 'copyTinymce'));
```
Comment (MitchCTiny) [inline modules/oxide/gulpfile.js:111]: Thanks for adding this, should make testing css changes between skins much faster

**Review by MitchCTiny — APPROVED** [review]

**Review by spocke — APPROVED** [review]

**Review by vpyshnenko — APPROVED** [review]

**Review by TheSpyder — APPROVED** [review]

File: modules/tinymce/CHANGELOG.md [hunk 3]
```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Templates will be parsed before preview and insertion to make preview consistent with inserted template content and prevent XSS. #TINY-9244
 - Pressing backspace in an empty line now preserves formatting in a previous empty line. #TINY-9454
 - Pressing enter inside the `inputfontsize` input would not move the focus back into the editor content. #TINY-9598
+- Improved contrast of table borders within dialogs in the `oxide-dark` skin. #TINY-9587
```
Comment (MitchCTiny) [inline modules/tinymce/CHANGELOG.md:30]:
```suggestion
- Improved contrast of border colors against the background color throughout the UI. #TINY-9587
```
Now that the scope has changed

**Review by MitchCTiny — APPROVED** [review]

**Review by tiny-ben-tran — APPROVED** [review]

**Comment by ArvinJ-H** [comment]
Commit profile error, reopen another one

**Comment by ArvinJ-H** [comment]
New PR
https://github.com/tinymce/tinymce/pull/8493

**Pull request closed without merging**